platform/engineering/design-docs/2020-12-10-auto-deploy-cms-build.md

```diff
@@ -0,0 +1,216 @@
+| Assets              | Uses the latest `vets-website` release and static asset                                                                                                                                                                                                                                                                                                                    | Creates a new release and deploys it                                                                                                                           |
+| Accessibility tests | N/A                                                                                                                                                                                                                                                                                                                                                                        | Runs tests                                                                                                                                                     |
+| Links checker       | Runs checks                                                                                                                                                                                                                                                                                                                                                                                                                                          | Runs checks                                                                                                                                                    |
+
+The VA.gov website can be deployed through the partial (or content-only) deploy or the full deploy.
+
+The full deploy is a daily job that deploys `vets-website` to the production environment of VA.gov. It uses the latest commit of `vets-website` as of 2:00pm ET on that day and the latest content downloaded from the Drupal server. It can be manually invoked as necessary, independently of the daily schedule.
+
+The content-only deploy builds `vets-website` with the latest content downloaded from the Drupal server and deploys to the dev and staging environments of VA.gov. This job can be triggered in one of two ways:
+- The daily production deploy of `vets-website`, which will build with latest release of `vets-website`
+- The `va-cms-bot`, which will build with the latest commit on `vets-website` in response to a manual trigger on the part of the CMS team
```

Comment:
The content release (content only deploy) should build with the previous release, not the latest commit.  Otherwise we would have unreleased code going out.

Reply:
I also think it would be good to spell out passing commits.   Because commits that have not passed would probably not be used in either type of deploy.